Formula/openshift-cli.rb

```diff
@@ -6,9 +6,9 @@ class OpenshiftCli < Formula
 
   stable do
     url "https://github.com/openshift/oc.git",
-        tag:      "openshift-clients-4.6.0-202006250705.p0",
-        revision: "51011e4849252c723b520643d27d3fa164d28c61"
-    version "4.6.0"
+        tag:      "openshift-clients-4.11.0-202204020828",
```

Comment:
It looks to me like the date part of the tag should be part of the version. Otherwise, we won't be able to ship newer tags that differ only in the date.

Reply:
worth a try on my end, edited manually :)

Reply:
I'll close it when I have a passing build, as so people coming in will be able to see the trial and error done :)